sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
+    
+    function __construct() {
+    }
+
+    private function get_settings($var_name, $default_value) {
+        if (isset($this->$var_name)) {
+            return $this->$var_name;
+        } else {
+            return $default_value;
+        }
+    }
+
+    function missing_person() {
```

Comment:
Creating different function for each configuration parameter will be a headache in future.

We can just create a function like `get('param-name')`. And take care of defaults somewhere else, probably in constructor.